homeassistant/components/aladdin_connect/cover.py

```diff
@@ -62,7 +61,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 class AladdinDevice(CoverEntity):
     """Representation of Aladdin Connect cover."""
 
-    def __init__(self, acc, device):
+    def __init__(self, acc: AladdinConnectClient, device: DoorDevice):
```

Comment:
```suggestion
    def __init__(self, acc: AladdinConnectClient, device: DoorDevice) -> None:
```

Reply:
why does mypy not complain about it :monocle_face:

Reply:
because for the init function it is assumed to return `None`. We are just more explicit 😉

Reply:
ahhh :smile: